--- Locale1.php	2019-12-04 23:54:10.856000000 +0300
+++ Locale2.php	2019-12-06 00:28:37.374281188 +0300
@@ -84,8 +84,9 @@
             }
 
             array_pop($localeSubTags);
+            $fallback = locale_compose($localeSubTags);
 
-            return locale_compose($localeSubTags);
+            return false !== $fallback ? $fallback : null;
         }
 
         if (false !== $pos = strrpos($locale, '_')) {
